Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<Project>
+    <PropertyGroup>
+        <LibplanetVersion>5.3.2-alpha.1</LibplanetVersion>
+
+        <!-- Fill with Libplanet's absolute path to debug with local Libplanet. -->
```

Comment:
```suggestion
        <!-- Fill with Libplanet's absolute path to debug with local Libplanet.
             Example: $(MSBuildThisFileDirectory).Libplanet -->
```
props are referenced by relative paths, so it is necessary to allow absolute paths to be specified.